### PR TITLE
DDF-3932 Fixed security-migratable permissions and made sure it would be started

### PR DESCRIPTION
**Pull request by paouelle**
#### What does this PR do?
Fixed a permission mistake where the etc directory was duplicated.
Also modified the security feature to ensure that the security migratable would be started.

#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
@brjeter 
@jhunzik 
@ethantmanns 

#### Select relevant component teams: 
https://github.com/orgs/codice/teams

#### Ask 2 committers to review/merge the PR and tag them here. If you don't know who to ask, you can request reviews in https://groups.google.com/forum/#!forum/ddf-developers .
(please choose ONLY two committers from below, delete the rest)
@figliold
@coyotesqrl 

#### How should this be tested? (List steps with links to updated documentation)
**This should be tested with a downstream distribution**
- Unzip and perform a migration:export
- Copy the 3 generated files somewhere safe
- Go through the installation
- Perform a migration:export
- Copy the 3 generated files somewhere safe
- Copy back the files exported the first time around and perform a migration:import --profile
- Verify through the UI that the system is back to install time
- Copy back the 3 files exported from the second time and perform a migration:import --profile
- Verify the system is back to a fully installed state

#### What are the relevant tickets?
[DDF-3932](https://codice.atlassian.net/browse/DDF-3932)

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: features/security/src/main/feature/feature.xml [hunk 1]
```diff
@@ -142,6 +142,14 @@
         <bundle>mvn:ddf.security.handler/security-handler-api/${project.version}</bundle>
     </feature>
 
+    <feature name="security-migratable" version="${project.version}"
+             description="Exports security configurations and system files">
+        <feature>guava</feature>
+        <bundle>mvn:ddf.platform.migration/platform-migratable-api/${project.version}</bundle>
+        <bundle>mvn:ddf.platform.api/platform-api/${project.version}</bundle>
```
Comment (brjeter) [inline features/security/src/main/feature/feature.xml]: ❓ Is there a reason you didn't just add the `<feature>platform-migratable-api</feature>` dependency which starts both of the above bundles?

Comment (paouelle) [reply features/security/src/main/feature/feature.xml]: circular dependencies

**Comment by paouelle** [comment]
build now

**Comment by ethantmanns** [comment]
Manually verified the changes ✅

**Pull request merged**